Makefile

```diff
@@ -2,6 +2,24 @@ LOG_LEVEL ?= spin=trace
 CERT_NAME ?= local
 SPIN_DOC_NAME ?= new-doc.md
 
+ARCH = $(shell uname -p)
+
+## dependencies for e2e-tests
```

Comment:
We might consider moving e2e-related targets/setup (fermyon platform, spin up tests, etc) into its own Makefile, perhaps under `tests/`.  Could still invoke from the top level (this Makefile) via an include.  Non-blocking of course, just thinking out loud.

Reply:
aah, that is a good idea. made note of it in my todo. will get that done soon.